ansible/roles/local_restore/tasks/get_backup_host.yml

```diff
@@ -6,13 +6,17 @@
     file_type: directory
   register: backup_directories
 
+- name: Order backup directories
+  ansible.builtin.set_fact:
+    backup_directories_ordered: "{{ backup_directories.files | sort(attribute='path', case_sensitive=False,reverse=True) }}"
```

Comment:
Why do you explicitly specify `case_sensitive=False`? What would happen if we omitted that parameter?

Reply:
I guess, this: https://stackoverflow.com/questions/11585884/jinja-alphabetically-sort-a-list-with-mixed-case